jme3-core/src/main/java/com/jme3/system/Annotations.java

```diff
@@ -69,4 +69,18 @@ public class Annotations {
      */
     public @interface Internal {
     }
+
+    /**
+     * Annotation used for signing some {@link com.jme3.testable.Testable}s to be selectively launched
+     * by {@link com.jme3.util.TestableExecutor}.
+     *
+     * @see com.jme3.util.TestableExecutor#launch(String[], Object, String[])
+     * @see com.jme3.util.TestableExecutor#launch(String, Object, String[])
+     */
+    @Documented
+    @Retention(RetentionPolicy.RUNTIME)
+    @Target({ElementType.TYPE})
+    public @interface Test {
+        String[] signatures();
```

Comment:
I'm not sure signature is the right word here. It makes me thing of digital signatures or hashes. Would tags be a better name?

Similarly, are these tests as such or examples?

Reply:
@richardTingle Thanks for reviewing, 

> I'm not sure signature is the right word here. It makes me think of digital signatures or hashes. Would tags be a better name?
We can use the default `values()`, however i agree with `tags()` and `markers()` too, what do you think ?

> Similarly, are these tests as such or examples ?
I have converted a lot of jme3 examples on the `jme3test` package to test this new system, and this is one of them, the `Launcher` class is your main class, could be anything not related to the feature (just for demonstration).

BTW, i have added `getCurrentActiveTestable()` as a feature and converted the utility to a singleton in the last 5 commits, so the usage will change to : 
```java
        TestableExecutor.getInstance().launch(new String[] {"jme3test"}, "--No-Data--", signatures);
```

Reply:
@richardTingle Alright, i changed it to a more friendly name:
```java
@Documented
@Retention(RetentionPolicy.RUNTIME)
@Target({ElementType.TYPE})
public @interface TestableTags {
    String[] value();
}
```
Check it out on the new commits.